README.rst

```diff
@@ -1,44 +1,16 @@
-*fMRIPrep*: A Robust Preprocessing Pipeline for fMRI Data
+*fMRIPrep/rodents*: Analysis-grade BOLD fMRI data of rats
```

Comment:
I suppose once we get a mouse template, we can modify this to:
`Analysis-grade BOLD fMRI data of rats and mice` but this works until that time

Reply:
Okay I have stated that mice are planned to be supported, but not at the moment.